docs/user_guide/02_medrecord.md

```diff
@@ -153,81 +181,120 @@ When building a MedRecord, you may want to save it to create a persistent versio
+
+```{exec-literalinclude} scripts/02_medrecord_intro.py
 ---
 language: python
-lines: 38
+setup-lines: 1-45
+lines: 48
 ---
 ```
 
-It will print an overview over all grouped nodes in the MedRecord.
+:::{dropdown} Methods used in the snippet
 
-```
--------------------------------------------------------
-Nodes Group Count Attribute Info                     
--------------------------------------------------------
-Medications 2     Name      Values: Insulin, Warfarin 
-Patients    3     Age       min: 64                   
-                            max: 74                   
-                            mean: 70.00               
-                  Loc       Values: GER, USA          
-                  Sex       Values: F, M              
-US-Patients 2     Age       min: 72                   
-                            max: 74                   
-                            mean: 73.00               
-                  Loc       Values: USA               
-                  Sex       Values: M                 
--------------------------------------------------------
-```
-As shown, we have two groups of nodes - Patients and Medications - created when adding the nodes. Additionally, there’s a group called 'US-Patients' that we created. For each group of nodes, we can view their attributes along with a brief statistical summary, such as the minimum, maximum, and mean for numeric variables.
+- [`overview_nodes()`](medmodels.medrecord.medrecord.MedRecord.overview_nodes){target="_blank"} : Gets a summary for all nodes in groups and their attributes.
+:::
 
-We can do the same to get an overview over edges in our MedRecord by using the `print_edge_overview` method:
+As shown, we have two groups of nodes - Patients and Medications - created when adding the nodes. Additionally, there’s a group called _'US-Patients'_ that we created. For each group of nodes, we can view their attributes along with a brief statistical summary, such as the minimum, maximum, and mean for numeric variables.
 
-```{literalinclude} scripts/02_medrecord_intro.py
+We can do the same to get an overview over edges in our MedRecord by using the [`overview_edges()`](medmodels.medrecord.medrecord.MedRecord.overview_nodes){target="_blank"} method:
+
+```{exec-literalinclude} scripts/02_medrecord_intro.py
 ---
 language: python
-lines: 40
+setup-lines: 1-46
+lines: 50
 ---
 ```
 
+However, they need to belong to a group in order to show their attributes in the overview.
+
+```{exec-literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+setup-lines: 1-46
+lines: 52, 55
+---
```

Comment:
Should the comment be shown here? Feels a bit weird imo, as we don't have comments anywhere else

![image](https://github.com/user-attachments/assets/9fb5b731-36cd-49ca-8189-1dabe99ed69e)